src/renderer/services/bitcoin/ledger.ts

```diff
@@ -20,47 +20,42 @@ const retrieveLedgerAddress = (network: Network) =>
     catchError((error) => Rx.of(RD.failure(error)))
   ).subscribe(setLedgerAddressRD)
 
-export const createLedgerService = (): LedgerService => ({
-  ledgerAddress$,
-  retrieveLedgerAddress,
-  removeLedgerAddress: () => setLedgerAddressRD(RD.initial)
-})
-
-export const createLedgerTransferService = () => {
-  const { get$: ledgerTxRD$, set: setLedgerTxRD } = observableState<LedgerTxRD>(RD.initial)
+const { get$: ledgerTxRD$, set: setLedgerTxRD } = observableState<LedgerTxRD>(RD.initial)
 
-  const ledgerTx$ = (params: LedgerTxInfoParams): LedgerTxLD =>
-    FP.pipe(
-      Rx.from(createTxInfo(params)),
-      switchMap((ledgerTxInfo) => window.apiHDWallet.signTxInLedger('BTC', params.network, ledgerTxInfo)),
-      switchMap((signResult) =>
-        Either.isRight(signResult)
-          ? Rx.from(broadcastTx({ txHex: signResult.right, nodeUrl: params.nodeUrl, nodeApiKey: params.nodeApiKey }))
-          : Rx.from(() =>
-              Promise.reject({
-                ledgerErrorId: signResult.left
-              })
-            )
-      ),
-      map(RD.success),
-      startWith(RD.pending),
-      catchError(
-        (e): LedgerTxLD =>
-          Rx.of(
-            RD.failure({
-              msg: !e.ledgerErrorId && e.toString(),
-              errorId: ErrorId.SEND_LEDGER_TX,
-              ledgerErrorId: e.ledgerErrorId
+const ledgerTx$ = (params: LedgerTxInfoParams): LedgerTxLD =>
+  FP.pipe(
+    Rx.from(createTxInfo(params)),
+    switchMap((ledgerTxInfo) => window.apiHDWallet.signTxInLedger('BTC', params.network, ledgerTxInfo)),
+    switchMap((signResult) =>
+      Either.isRight(signResult)
+        ? Rx.from(broadcastTx({ txHex: signResult.right, nodeUrl: params.nodeUrl, nodeApiKey: params.nodeApiKey }))
+        : Rx.from(() =>
+            Promise.reject({
+              ledgerErrorId: signResult.left
             })
           )
-      )
+    ),
```

Comment:
BTW: You can use `E.fold` to avoid `Either.isRight(..) ? .. : ..`
```
    switchMap((signResult) =>
      FP.pipe(
        signResult,
        E.fold(
          (ledgerErrorId) =>
            Rx.from(
              Promise.reject({
                ledgerErrorId
              })
            ),
          (txHex) => Rx.from(broadcastTx({ txHex, nodeUrl: params.nodeUrl, nodeApiKey: params.nodeApiKey }))
        )
      )
    ),
```